src/library/Connect/WorkloadInformationProvider.cs

```diff
@@ -100,14 +103,14 @@ internal class WorkloadInformationProvider : IWorkloadInformationProvider
                         reachableEndpoints.Add(new EndpointInfo()
                         {
                             DnsName = externalEndpoint.Name,
-                            Ports = externalEndpoint.Ports.Select(p => new PortPair(remotePort: p)).ToArray(),
+                            Ports = externalEndpoint.Ports.Select(p => new PortPair(remotePort: p)).ToArray(), 

```

Comment:
Can we add comment here that we should not take into account ports to ignore here since this code path is handling endpoints user explicitly added for tracking.